pkg/clusterd/disk.go

```diff
@@ -84,20 +84,45 @@ func DiscoverDevices(executor exec.Executor) ([]*sys.LocalDisk, error) {
 			logger.Warningf("failed to get udev info for device %q. %v", d, err)
 		}
 
+		if disk.Type == sys.PartType {
```

Comment:
I'm not sure this is the right place to do the filtering, typically, this reports devices only. I know we have a precedent L66 but it shouldn't have been there. Normally the filtering happens in `getAvailableDevices()`. For consistency, I'd like to have this handled in `getAvailableDevices()` instead.
Also, if you do this, you don't need the double-check, you will just go over the child and only `PartitionIsAtari()` will be needed.

Reply:
I think both of your comments are really sound. I will opt to close this PR and create a new one to handle the gpt-only support since it is pretty much wholly different.